Modelica/Utilities/Examples.mo

```diff
@@ -282,6 +282,10 @@ expression(\"sin(pi/6)\");  // returns 0.5
                            loadSelector(filter="Text files (*.txt)",
                            caption="Open file in which Real parameters are present")));
     input String name "Name of parameter";
+    input Boolean useDefaultValue = false "If true, 
```

Comment:
Please avoid line breaks in description strings.

Reply:
I would leave it here since the string is veeery long anyway. So better have it broken in a natural place like a comma.